tests/test_client.py

```diff
@@ -146,3 +145,33 @@ def test_nanopub_publish_blanknode(self):
         pubinfo = client.publish(nanopub)
         assert pubinfo['nanopub_uri'] == test_published_uri
         assert pubinfo['concept_uri'] == expected_concept_uri
+
+    def test_retract_with_force(self):
+        client = NanopubClient()
+        client.java_wrapper.publish = mock.MagicMock()
+        client.retract('http://www.example.com/my-nanopub', force=True)
```

Comment:
I wonder if this is a "true" retraction in the eyes of Tobias, since you're supposed to have exactly the same public key to be able to retract a nanopub. I do agree that having a ```force=True``` option makes sense though.

Reply:
Yeah, good point...